src/internal/connector/exchange/service_iterators.go

```diff
@@ -317,20 +314,19 @@ func FetchMessageIDsFromDirectory(
 		deltaToken string
 	)
 
-	options, err := optionsForFolderMessages([]string{"id"})
+	options, err := optionsForFolderMessagesDelta([]string{"id"})
 	if err != nil {
 		return nil, deltaToken, errors.Wrap(err, "getting query options")
 	}
-
+	//*msuser.UsersItemMailFoldersItemMessagesDeltaRequestBuilder
```

Comment:
Is this comment needed?

Reply:
Resolved here: [167803e](https://github.com/alcionai/corso/pull/1479/commits/167803eb06d9a57d870eea557851c783c1f3c0b8)